src/App.js

```diff
@@ -2,11 +2,19 @@ import { ThemeProvider } from 'styled-components';
 import './App.css';
 import Store from './components/Store';
 import theme from './theme';
+import styled from 'styled-components';
+import css from '@styled-system/css'
+
+const Button = styled.button`
+  background-color: ${({theme}) => (theme.colors.purple)};
+`
 
 function App() {
   return (
     <ThemeProvider theme={theme}>
       <Store/>
+      <Button css={css({backgroundColor: theme.colors.darkBlue})}>Test Button1</Button>
```

Comment:
Since we're inside the `ThemeProvider`, we should be able to use the name of the color in the theme directly instead of accessing the color from the `theme` object. What do you think?